codespell_lib/data/dictionary.txt

```diff
@@ -19118,6 +19142,15 @@ unsable->unusable, usable, unstable,
 unsccessful->unsuccessful
 unsearcahble->unsearchable
 unsed->unused, used,
+unselcted->unselected
+unselctes->unselects
+unselcting->unselecting
+unselcts->unselects
+unselecgt->unselect
+unselecgted->unselected
+unselecgtes->unselects
+unselecgting->unselecting
+unselecgts->unselects
 unselect->deselect
```

Comment:
Travis is failing because of this. See also here:
https://english.stackexchange.com/questions/18465/unselect-or-deselect

Reply:
I see this come up very often and i have learned to ignore it. Not sure what to do here.

Reply:
Well we either need to remove `unselect->deselect` as a correction, or make all the ones that suggest unselect actually suggest deselect.

Reply:
I'd say get rid of `unselect->deselect` since it's context sensitive grammar. All the others are typos.

Reply:
Sounds good to me, do you want to do the honours. That thread is a bit vague either way, but there seem to be enough instances of it, and reasons for it, that it doesn't seem unreasonable.